Runtime/Shaders/ComputeShaderHelper.cs

```diff
@@ -36,6 +36,10 @@ internal class ComputeShaderHelper {
             };
 
             renderTarget.Create();
+            
+#if UNITY_EDITOR
```

Comment:
Why is it `UNITY_EDITOR` only ?
Don't we have the same memory leak in runtime ?

Reply:
I described the issue above in the other comment.